pkg/meshctl/commands/dashboard/dashboard.go

```diff
@@ -0,0 +1,157 @@
+package dashboard
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/browser"
+	"github.com/solo-io/gloo-mesh/pkg/meshctl/utils"
+	"github.com/solo-io/skv2/pkg/multicluster/kubeconfig"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func Command(ctx context.Context) *cobra.Command {
+	opts := &options{}
+	cmd := &cobra.Command{
+		Use:   "dashboard",
+		Short: "Port forwards the Gloo Mesh Enterprise UI and opens it in a browser if available",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return forwardDashboard(ctx, opts.kubeconfig, opts.kubecontext, opts.kubens, opts.port)
+		},
+	}
+	opts.addToFlags(cmd.Flags())
+
+	cmd.SilenceUsage = true
+	return cmd
+}
+
+type options struct {
+	kubeconfig  string
+	kubecontext string
+	kubens      string
+	port        uint32
+}
+
+func (o *options) addToFlags(flags *pflag.FlagSet) {
+	utils.AddManagementKubeconfigFlags(&o.kubeconfig, &o.kubecontext, flags)
+	flags.StringVar(&o.kubens, "kubes-namespace", "gloo-mesh", "The namespace that the Gloo Mesh UI is deployed in")
+	flags.Uint32VarP(&o.port, "port", "p", 8090, "The local port to forward to the dashboard")
+}
+
+var ConsoleNotFoundError = errors.New("Console image not found. Your Gloo Mesh enterprise install may be a bad state.")
+
+func forwardDashboard(ctx context.Context, kubeconfigPath, kubectx, kubens string, localPort uint32) error {
```

Comment:
nit, since "namespace" implies k8s:

```suggestion
func forwardDashboard(ctx context.Context, kubeconfigPath, kubectx, namespace string, localPort uint32) error {
```